config/src/lib.rs

```diff
@@ -177,36 +177,53 @@ impl Default for ConsensusAPIGrpcParameters {
     }
 }
 
+fn block_synchronizer_params_default_value() -> Duration {
```

Comment:
Would it be possible to avoid using block_synchronizer_params_default_value, and rather have 4 distinct default values (which can be consts defined in an `impl BlockSynchronizerParameters` block), one for each timeout, referred to on both the the serialize and the implementation of `Default`?

Reply:
Agree that we may need to adjust these values independently, although I don't have a good sense of the default to use other than 30s for now. Changed to separate default values.